pom.xml

```diff
@@ -66,6 +66,8 @@ Portions Copyright (c) 2018, Chris Fraire <cfraire@me.com>.
         <compileTarget>1.8</compileTarget>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jersey.version>2.27</jersey.version>
+        <!-- Jackson version needs to match the version of Jackson Jersey depends on otherwise weird things will happen. -->
+        <jackson.version>2.8.10</jackson.version>
```

Comment:
maybe you can use exclude for deps like this to be sure whats happening.

Reply:
Not sure if you can use dynamic dependencies (version wildcard, in this case something like 2.9.*) in Maven. It would be better to enforce this positively - somehow extract the Jackson version Jersey depends on and make it explicit dependency.

I did not really want to have a implicit dependency however maybe that is more robust than this - the risk of Jersey to be removed is fairly low (and the `Statistics` tests would break anyway).

Reply:
Yes ok. Maybe you could use some maven plugin to extract the version but it leads to complex solution to not-really-a-problem thing. So I would stick with this.

Reply:
The same question was asked on https://stackoverflow.com/questions/23891067/maven-access-version-of-transitive-dependency-via-property however received no answer so far (4 years).

Reply:
https://howtodoinjava.com/maven/maven-dependency-management/ spells out this problem nicely:
> Transitivity brings a very serious problem when different versions of the same artifacts are included by different dependencies. It may cause version mismatch issue in runtime.

It is a bit surprising that Maven (or rather the standard plugins) does not have a solution built in.